docs/essentials/exceptions/exceptions.md

```diff
@@ -1,21 +1,21 @@
 # Handling and throwing exceptions
 
-Applications must be able to handle errors that occur during execution in a consistent manner. The common language runtime provides a model for notifying applications of errors in a uniform way. All .NET Framework operations indicate failure by throwing exceptions.
+Applications must be able to handle errors that occur during execution in a consistent manner. The common language runtime (CLR) provides a model for notifying applications of errors in a uniform way. All .NET Core operations indicate failure by throwing exceptions.
```

Comment:
We should avoid "CLR" and ".NET Core" terms. We should say "runtime" and ".NET".